src/apps/content/templates/support/about_us.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+{% block title %}
```

Comment:
Where title block is defined? Can you prove to me that it exists?

Reply:
to be honest asked chat gpt your previous question and got this, not sure, will read about it on weekend and fix/prove on Monday)

Reply:
Ok i think i have figure it out now i am registrating this block in base.html and redefine it it other templates. I also using h1 instead of title cos in my opinion it looks better that way. I saw the meta block in header but it looks very ugly with it ( the title of the page is in the wrong place if i am using it)

Reply:
great!